src/screens/tests/RestrictedScreen.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { InjectedIntlProps } from 'react-intl';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { rootReducer } from '../../modules';
+import { RestrictedScreen } from '../RestrictedScreen';
+
+const store = createStore(rootReducer);
+
+const setup = (props: Partial<InjectedIntlProps> = {}) =>
+    shallow(
+        <Provider store={store}>
+            <RestrictedScreen />
+        </Provider>,
+    );
+
+describe('RestrictedScreen', () => {
+    const wrapper = setup();
+
+    it('should render', () => {
+        expect(wrapper).toMatchSnapshot();
```

Comment:
What is a reason to add tests like this? It makes no sense, only match snapshot😂